ddtrace/appsec/internal/sqreen.py

```diff
@@ -27,7 +28,9 @@ class SqreenLibrary(BaseProtection):
     def __init__(self, rules, budget_ms=None):
         # type: (str, Optional[float]) -> None
         if budget_ms is None:
-            budget_ms = DEFAULT_BUDGET_MS
+            budget_ms = float(
+                get_env("appsec", "waf_budget_ms", default=DEFAULT_WAF_BUDGET_MS)  # type: ignore[arg-type]
```

Comment:
👍🏻 to `waf_budget_ms`

Reply:
why the ignore arg-type? because of `float()` ?

Reply:
The return type of `get_env()` is `Optional[str]` so the argument type of `float()` is incorrect. In practice, it always returns a string when `default` is set though.